protoc-gen-twirp_ruby/main.go

```diff
@@ -74,13 +74,14 @@ func (g *generator) generateFile(file *descriptor.FileDescriptorProto) *plugin.C
 		serviceName := serviceName(service)
 		g.P(`require 'twirp'`)
 		g.P(``)
-		g.P(fmt.Sprintf("class %sService < Twirp::Service", serviceName))
+		g.P(fmt.Sprintf("class %s::%sService < Twirp::Service", CamelCase(pkgName), serviceName))
```

Comment:
This will definitely not work because the generator should handle protobuf files with no package, and packages with multiple dots (e.g. `package my.nested.package`).

This doesn't need to be fixed in this PR though, we're not in a stable version yet.